doc/src/manual/noteworthy-differences.md

```diff
@@ -224,6 +229,23 @@ For users coming to Julia from R, these are some noteworthy differences:
   * The commonly used `Int` type corresponds to the machine integer type (`Int32` or `Int64`).
     This means it will overflow, such that `2^64 == 0`. If you need larger values use another appropriate type,
     such as `Int128`, [`BigInt`](@ref) or a floating point type like `Float64`.
+  * The imaginary unit `sqrt(-1)` is represented in Julia as `im`, not `j` as in Python.
+  * In Julia, the exponentiation operator is `^`, not `**` as in Python.
+  * Julia uses `nothing` of type `Nothing` to represent a null value, whereas Python uses `None` of type `NoneType`.
+  * In Julia, the standard operators over a matrix type are matrix operations, whereas, in Python, the standard operators are element-wise operations. When both `A` and `B` are matrices, `A * B` in Julia performs matrix multiplication, not element-wise multiplication as in Python. `A * B` in Julia is equivalent with `A @ B` in Python, whereas `A * B` in Python is equivalent with `A .* B` in Julia.
+  * The adjoint operator `'` in Julia returns an adjoint of a vector (a lazy representation of row vector), whereas the transpose operator `.T` over a vector in Python returns the original vector (non-op).
+  * Functions in Python have one implementation (no polymorphism), whereas in Julia a function may contain multiple concrete implementations (called *Methods*), selected via multiple dispatch.
```

Comment:
As a reader, I have found it helpful to keep this list entirely in the pattern of "Julia behavior in contrast to other language behavior."

Reply:
I aligned the pattern as you suggested for most of the points.
A few points do not fit so well content-wise into this pattern (e.g. showing how methods of Python objects translate to Julia), therefore I kept them as they are. 
Could you please check if it is fine for you now?

Reply:
Yes, that's awesome, thanks!  I know it's not always possible/readable that way, this is much easier for me to read. 👍